modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -208,7 +208,7 @@ components:
                 externalReferenceCode:
                     type: string
                 type:
-                    enum: [AssetLibrary, Site]
+                    enum: [AssetLibrary, CMS, Site]
```

Comment:
This schema is shared with the `FileEntry` entity too, and I think in the `FileEntry` this is write-read. So I am worried about creating a File Entry in the CMS scope. I think that should be forbidden as part of this story